dbclient/mongodb/src/main/java/io/helidon/dbclient/mongodb/MongoDbClientProviderBuilder.java

```diff
@@ -66,9 +67,13 @@ public DbClient build() {
 
     @Override
     public MongoDbClientProviderBuilder config(Config config) {
-        config.get("url").asString().ifPresent(this::url);
-        config.get("username").asString().ifPresent(this::username);
-        config.get("password").asString().ifPresent(this::password);
+        config.get("connection").asNode().ifPresentOrElse(conn -> {
+            conn.get("url").asString().ifPresent(this::url);
+            conn.get("username").asString().ifPresent(this::username);
+            conn.get("password").asString().ifPresent(this::password);
+        }, () -> {
+            throw new DbClientException("No database connection configuration was found");
```

Comment:
You should mention the exact key that is expected. Use `config.get("connection").key()`

Reply:
Done.